.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
i would leave this out which means raqm would not be tested by default.  you should submit the library for inclusion into places first.